### DISCUSSION TITLE
Stop background scrolling when Modal is open

### DISCUSSION THREAD

**Discussion by compilekaiten** [saadeghi/daisyui#1240] [Q&A]
Using Vue3/Vite.

I have multiple responsive modals from a `v-for`.

How can I stop the background from being scrolled when the modal is open.

**Comment by fini** [comment]
In my react implementation, I change body overflow css style on mount and unmount of the modal component:

```
// Make sure the body is not scrollable, while modal is shown.
  useMount(() => {
    document.body.style.overflow = 'hidden'
    return () => {
      document.body.style.overflow = 'auto'
    }
  })
```
